vaccine/tableau.cpp

```diff
@@ -116,12 +132,13 @@ namespace vaccine {
                 }
               } catch (...) {
                 DLOG_F(ERROR, "Exception");
+                statusCode = 500;
               }
 
             }
 
           } else {
-            statusCode = 404;
+            statusCode = (method == kREQUEST_GET) ? 200 : 404;
```

Comment:
I think the whole logic is bad. if we get into the with_object closure then it should be 200, otherwise 404